app/Filament/Resources/ServerResource.php

```diff
@@ -24,22 +32,94 @@
 use Filament\Tables\Table;
 use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Database\Eloquent\SoftDeletingScope;
-use phpDocumentor\Reflection\Types\Self_;
+
 
 class ServerResource extends Resource
 {
     public const ICON = 'tabler-server-2';
 
     protected static ?string $model = Server::class;
-
     protected static ?string $navigationIcon = self::ICON;
+    protected static ?string $navigationLabel = 'Runners';
 
+    protected static ?string $modelLabel = 'Runners';
     protected static ?int $navigationSort = 30;
 
     public static function form(Form $form): Form
     {
         return $form
-            ->schema(Server::getForm());
+            ->schema([
+                FormSection::make('Server Information')
+                    ->columns(2)
+                    ->icon(ServerResource::ICON)
+                    ->schema([
+                        Select::make('server_type')
+                            ->options(
+                                collect(
+                                    ServerType::cases())
+                                    ->mapWithKeys(fn($type) => [$type->value => $type->label()])
+                            )
```

Comment:
according to the docs i think you can just do
```suggestion
                            ->options(ServerType::class)
```

reference: https://filamentphp.com/docs/3.x/support/enums